web/src/components/building/GarbageSchedule.vue

```diff
@@ -27,7 +27,7 @@
             class="garbage-card"
           >
             <div class="spread-out">
-              <p class="garbage-type">{{ item.action.description }}</p>
+              <p class="garbage-type">{{ "test" }}</p>
```

Comment:
Dit component is aangepast, maar wordt helemaal niet gebruikt?

Reply:
Dacht dat ik deze verwijderd had, my bad, ik haal hem er nu uit.